### PR TITLE
feat: add auditlogs for account model

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

- Add audit logs for the Account model

image

Fixes https://linear.app/chatwoot/issue/CW-1762/update-the-display-text-account-activity-logs
## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Tested locally by updating the account-related configuration
- Added specs

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: enterprise/app/models/enterprise/audit/account.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ module Enterprise::Audit::Account
   extend ActiveSupport::Concern
 
   included do
+    audited
```
Comment (sojan-official) [inline enterprise/app/models/enterprise/audit/account.rb:5]: let's limit this to update alone for now.

File: app/javascript/dashboard/helper/auditlogHelper.js [hunk 2]
```diff
@@ -30,6 +30,9 @@ const translationKeys = {
   'accountuser:create': `AUDIT_LOGS.ACCOUNT_USER.ADD`,
   'accountuser:update:self': `AUDIT_LOGS.ACCOUNT_USER.EDIT.SELF`,
   'accountuser:update:other': `AUDIT_LOGS.ACCOUNT_USER.EDIT.OTHER`,
+  'account:create': `AUDIT_LOGS.ACCOUNT.ADD`,
+  'account:update': `AUDIT_LOGS.ACCOUNT.EDIT`,
+  'account:destroy': `AUDIT_LOGS.ACCOUNT.DELETE`,
```
Comment (sojan-official) [inline app/javascript/dashboard/helper/auditlogHelper.js:35]: lets limit this to update alone for now

**Review by sojan-official — COMMENTED** [review]
Minor comment on the associated events. 
code lgtm other wise

**Comment by vishnu-narayanan** [comment]
- Audit on the only account update event

File: enterprise/app/models/enterprise/audit/account.rb (hunk 1 above)
Comment (vishnu-narayanan) [reply enterprise/app/models/enterprise/audit/account.rb]: done

File: app/javascript/dashboard/helper/auditlogHelper.js (hunk 2 above)
Comment (vishnu-narayanan) [reply app/javascript/dashboard/helper/auditlogHelper.js:35]: done

**Review by sojan-official — APPROVED** [review]

**Pull request merged**